test-result-summary-client/src/utils/Utils.js

```diff
@@ -2,6 +2,15 @@ const order = (a, b) => {
     const collator = new Intl.Collator(undefined, { numeric: true, sensitivity: 'base' });
     return collator.compare(a, b);
 }
+const getInfoFromBuildName = (buildName) => {
+    const regex = /^Test_openjdk(\w+)_(\w+)_(\w+).(.+?)_(.+?_.+?(_xl)?)(_.+)?$/i;
+    const tokens = buildName.match(regex);
+    let platform;
+    if (Array.isArray(tokens) && tokens.length > 5) {
+        platform = tokens[5];
+    }
+    return platform;
+}

```

Comment:
I meant to create a function so that we can use it in multiple places. It is not only for TabularView.jsx, but also for [ResultSummary.jsx
](https://github.com/AdoptOpenJDK/openjdk-test-tools/blob/31803c2e1a4772db48a711006d6b1e29a922d237/test-result-summary-client/src/Build/Summary/ResultSummary.jsx#L56)

And this function should return all info, not only the platform.